tests/expr/t08_if_only_body_unit.test

```diff
@@ -0,0 +1,3 @@
+discard """
+"""
```

Comment:
```suggestion
  output: "(): unit" 
"""
```
To make sure that the type is computed correctly, and that there are no run-time errors.